### ISSUE TITLE
[Bug]: "Failed to load community plugins."

### ISSUE THREAD

**Issue by ivan3-5** [denolehov/obsidian-git#527]
### Describe the bug

When I enable obsidian git, it fails to load community plugins when I want to browse for community plugins and the same goes for the community themes too as well.

### Steps to reproduce

*Enabling obsidian git plugin makes community themes and plugins don't load in browse menu.

### Operating system

Windows

### Installation Method

Other

### Plugin version

2.20.2

**Comment by ivan3-5** [comment]
> What exactly happens? Does it simply load infinitely long? Does it work when you disable the plugin again? If not, does it work after a restart of Obsidian and the plugin turned off?

I just come back from noting on obsidian(since I don't note a lot). I turned off the plugin and now the obsidian just worked fine and I can browse plugins that I haven't installed now the same goes for themes too. I tried updating the plugin

**Comment by ivan3-5** [comment]
> What exactly happens? Does it simply load infinitely long? Does it work when you disable the plugin again? If not, does it work after a restart of Obsidian and the plugin turned off?

I just came back taking notes on obsidian(since I don't note a lot). I turned off the plugin and the obsidian just worked fine and I can browse plugins that I haven't installed now the same goes for themes too. I tried updating the plugin and restarting obsidian after enabling the plugin. And it works fine now. I don't know what happened but updating the plugin fixed the issue. Thank you btw. Sorry for tge trouble.
